backend/tests/unit_tests/config/payloads/payloads.json

```diff
@@ -58,7 +58,58 @@
         }
       }
     },
-    "read": {},
+    "read": {
+      "test_read_w_key": {
+        "config_definition_key": "sample_config_definition",
+        "config_key": "sample_config",
+        "schema": {
+          "json_schema": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "date": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "name",
+              "date"
+            ]
+          }
+        },
+        "return_value": {
+          "rows_affected": 0, 
+          "response": [{"data": {"name": "test_read_w_key", "date": "2018-01-01"}}]
+        }
+      },
+      "test_read_o_key": {
+        "config_definition_key": "sample_config_definition",
+        "config_key": "",
+        "schema": {
+          "json_schema": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "date": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "name",
+              "date"
+            ]
+          }
+        }
+      },
+      "test_read_o_cd_key": {
+        "config_definition_key": "",
+        "config_key": "sample_config"
+      }
+    },
```

Comment:
@coderabbitai Thank you for the suggestion, but I can explain the workings of the tests, so that we can get a better standpoint at this.

> Missing return_value fields in test_read_o_key and test_read_o_cd_key test cases

The `return_value` is missing in the `o_key` and `o_cd_key` tests, because these two tests are supposed to fail. It translates to `without configuration key`, `without configuration definition key`.

Which also solves the confusion which might arise about not having a test for non-existent configuration keys as both of them handle the same.

> No test cases for invalid data that fails schema validation

The `schema` field is just used to mock the return of a config definition key's read. Meaning, this should and must only exist for the tests which have a successful config definition read - `o_key` and `w_key`.